app/helpers/orchestration_stack_helper/textual_summary.rb

```diff
@@ -56,11 +56,12 @@ def textual_ems_cloud
   def textual_orchestration_template
     template = @record.orchestration_template
     return nil if template.nil?
+    return nil unless template.orderable
     label = ui_lookup(:table => "orchestration_template")
     h = {:label => label, :image => "orchestration_template", :value => template.name}
     if role_allows(:feature => "orchestration_templates_view")
       h[:title] = _("Show this Orchestration Template")
-      h[:link] = url_for(:controller => 'catalog', :action => 'ot_show', :id => template.id)
+      h[:link] = url_for(:action => 'show', :id => @orchestration_stack, :display => 'stack_orchestration_template')
```

Comment:
@h-kataria @mzazrivec either way is acceptable to me. If always showing the new screen, the user experience is the same for both orderable and non-orderable. If jumping to Catalog explorer, it makes the management of all templates easier. I personally prefer the new screen for both cases.

@mzazrivec the Configuration menu `Show this Orchestration Template in a Catalog`, should it be `Show this Orchestration Template in Catalogs`?